src/main/java/ch/uzh/ifi/hase/soprafs22/entity/User.java

```diff
@@ -108,5 +108,22 @@ public void setStatus(UserStatus status) {
 
     public Gender getGender(){return this.gender; }
 
+    /**
+     * Method for Testing the gender of the user in JSON, as the gender should be
+     * as string and not ENUM there
+      * @return User.gender as string
+     */
+    public String getGenderString(){
```

Comment:
![image](https://user-images.githubusercontent.com/75019468/162150743-5c8b7d5f-108d-4ec5-8060-544e3bd40387.png)
Wouldn't that make it much easier?

Reply:
Will make changes, forgot you could do that

Reply:
If done tests stop passing, not sure why, propose to leave it as it is?

Reply:
In UserGetDTO, change getGender return type back to Gender, then it should work (atleast for the tests)

Reply:
Ahh nice, thanks, works now.